### PR TITLE
Add work in progress files for Fr translation

### PR DESCRIPTION
**Pull request by angelique-dubois-lrn**
@shtrom @SalimBensiali

### CODE REVIEW

**Comment by angelique-dubois-lrn** [comment]
Hello @shtrom , 
I've managed to translate a few more. I think it'd be great if you can have a look at my translations, some of them might use a reread, and maybe optimisation. I did google a lot of math/chemistry terms, and used my (very) old memories from high school ^^'

From the last version I uploaded, below my main comments:

Questions API

Line 180 - poor translation?
"submitComplete": "Submitted”,
"submitComplete": "Soumis(e)(s)”, 

Line 394 - not translated (not sure how to translate... Is it "angle"?)
"overarc": "Arc”,  

Lines 403 to 406 - not translated. Not sure how to translate here either.
        "overleftrightarrow": "Line",
        "overline": "Segment",
        "dot": "Recurring Decimal",
        "longdiv": "Long division”, 

Lines 465, 466 - Not translated. 
"rightleftharpoons": "Right / left harpoons”,
"leftrightharpoons": "Left / right harpoons”,

Lines 500, 501: 
How do we handle decimal/comma for French users?
I translated literally, but if we use comma for decimal and point for thousands, we'll need to change the translation.


Assess API

Line 128,129 - not translated.
"discardInProgress": "Exiting…",
"discardError": "Exit &amp; Discard failed.",

**Comment by shtrom** [comment]
Hey, sorry for the delay in getting back to you.

> Line 180 - poor translation?
> "submitComplete": "Submitted”,
> "submitComplete": "Soumis(e)(s)”,

Are we using `Activité` the rest of the time? Maybe `Soumise`. Or `Soumission complète`

> Line 394 - not translated (not sure how to translate... Is it "angle"?)
> "overarc": "Arc”,
> 
> Lines 403 to 406 - not translated. Not sure how to translate here either.
> "overleftrightarrow": "Line",
> "overline": "Segment",
> "dot": "Recurring Decimal",
> "longdiv": "Long division”,
> 
> Lines 465, 466 - Not translated.
> "rightleftharpoons": "Right / left harpoons”,
> "leftrightharpoons": "Left / right harpoons”,

I think for a lot of those hard-to-translate terms, they are buttons for the calculator. We'd need to track down all the available buttons, and see what the sign actually is. I think that would help us find the correct translation.

> Lines 500, 501:
> How do we handle decimal/comma for French users?
> I translated literally, but if we use comma for decimal and point for thousands, we'll need to change the translation. 

I think we should use proper French notation, as you describe, but it might be tricky, as we'd need to extend that to code that formats numbers in the rest of the product(which I'm not sure whether we have or not).

> Line 128,129 - not translated.
> "discardInProgress": "Exiting…",
> "discardError": "Exit & Discard failed.",

`Abandon en cours...`?
`Erreur lors de l'abandon`?

**Comment by angelique-dubois-lrn** [comment]
Hi @shtrom ,

> I think for a lot of those hard-to-translate terms, they are buttons for the calculator. We'd need to track down all the available buttons, and see what the sign actually is. I think that would help us find the correct translation.

Yes I did this for most of the translation. Also used the latex to find more info and more accurate translation possible using https://fr.wikipedia.org/wiki/Table_de_symboles_math%C3%A9matiques for example.
For those I'm not sure because I would translate Line and Segment by `droite` and `segment`, but the button in the calculator doesn't remind me of a notation I have seen before for these.

> I think we should use proper French notation, as you describe, but it might be tricky, as we'd need to extend that to code that formats numbers in the rest of the product(which I'm not sure whether we have or not).

Do you know who I can reach out to find out this ? 


Thanks

**Comment by shtrom** [comment]
> For those I'm not sure because I would translate Line and Segment by droite and segment

I wonder if those two are actual geometrical objects, in which case you'd be right, or math notations (i.e., over variable names or symbols), in which case I don't know what to call them...

Edit: yeah, reviving my LaTeX, all the `\over` something are math notations, not geometrical objects.

**Comment by shtrom** [comment]
https://kapeli.com/cheat_sheets/LaTeX_Math_Symbols.docset/Contents/Resources/Documents/index

**Comment by shtrom** [comment]
AH! https://mirror.aarnet.edu.au/pub/CTAN/info/lshort/french/lshort-fr.pdf

> Les commandes \overline et \underline créent un trait horizontal au-dessus ou au-dessous d’une expression

Not everything has ready translations, but there are examples, and partial names, so we can probably play from there, e.g.,

* `\overleftrightarrow` -> “double flèche” with variations around ”accent/au-dessus/...”

File: languages/fr/questions-api.json [hunk 1]
```diff
@@ -398,11 +398,11 @@
...
-        "overrightarrow": "Vecteur flèche vers la droite",
-        "overleftarrow": "Vecteur flèche vers la gauche",
-        "overleftrightarrow": "Line",
-        "overline": "Segment",
-        "dot": "Recurring Decimal",
+        "overrightarrow": "Flèche vers la droite au-dessus",
+        "overleftarrow": "Flèche vers la gauche au-dessus",
+        "overleftrightarrow": "Flèche bilatérale au-dessus",
+        "overline": "Trait au-dessus",
+        "dot": "Point au-dessus",
```
Comment (shtrom) [inline languages/fr/questions-api.json]: 👍

**Review by shtrom — APPROVED** [review]
I think this is a good start. Could we perhaps remove all non-translated strings, if any (so we can `grep` for them; might also be worth keeping a list of them to remember), and then merge this?

**Comment by angelique-dubois-lrn** [comment]
HI @shtrom ,
thanks

I updated both files.

I have the list of non translated lines. Not sure how/where to document it though, any guideline or idea? Thanks

**Comment by shtrom** [comment]
Maybe just a MISSING_TRANSLATIONS file in this directory, with the name of one of the strings per line.

**Pull request merged**